.github/workflows/linux.yml

```diff
@@ -103,6 +104,10 @@ jobs:
       - runner-select
     name: Linux Build [${{ needs.runner-select.outputs.unique-id }}]
     runs-on: ${{ needs.runner-select.outputs.selected-runner-label }}
+    permissions:
```

Comment:
Can the permissions be declared on the parent workflow, like we do for [android](https://github.com/servo/servo/blob/dd9164f49a3f53bd4567687b009da6df9ffd11ed/.github/workflows/nightly.yml#L125) and ohos?